Description: Use local javadoc links to avoid a build failure with Java 10
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/gradle/docs.gradle
+++ b/gradle/docs.gradle
@@ -33,9 +33,9 @@ def javadocSpec = {
         overview = rootProject.file('src/main/overviewj.html')
         footer = doc.footer
         source = rootProject.useIndy()?'1.7':'1.6'
-        links('http://docs.oracle.com/javase/8/docs/api/', 'http://docs.oracle.com/javaee/7/api/',
-                'http://commons.apache.org/proper/commons-cli/javadocs/api-release/', 'http://junit.org/junit4/javadoc/latest/',
-                'http://docs.oracle.com/javaee/6/api/', 'http://www.antlr2.org/javadoc/')
+        links('file:///usr/share/doc/ant/api/', 'file:///usr/share/doc/default-jdk/api/',
+              'file:///usr/share/doc/libservlet3.1-java-doc/api',
+              'file:///usr/share/doc/junit4/api/')
     }
 }
 
@@ -53,12 +53,10 @@ def groovydocSpec = {
             overviewText = resources.text.fromFile(rootProject.file('src/main/overview.html'))
         }
         includePrivate = false
-        link 'http://docs.oracle.com/javaee/7/api/', 'javax.servlet.', 'javax.management.'
-        link 'http://docs.oracle.com/javase/8/docs/api/', 'java.', 'org.xml.', 'javax.', 'org.w3c.'
-        link 'http://docs.groovy-lang.org/docs/ant/api/', 'org.apache.ant.', 'org.apache.tools.ant.'
-        link 'http://junit.org/junit4/javadoc/latest/', 'org.junit.', 'junit.'
-        link 'http://www.antlr2.org/javadoc/', 'antlr.'
-        link 'http://commons.apache.org/proper/commons-cli/javadocs/api-release/', 'org.apache.commons.cli.'
+        link 'file:///usr/share/doc/libservlet3.1-java-doc/api', 'javax.servlet.', 'javax.management.'
+        link 'file:///usr/share/doc/default-jdk/api/', 'java.', 'org.xml.', 'javax.', 'org.w3c.'
+        link 'file:///usr/share/doc/ant/api/', 'org.apache.tools.ant.'
+        link 'file:///usr/share/doc/junit4/api/', 'org.junit.'
 }
 
 allprojects {
